client/react/containers/BrowseListings.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import { Button, Card, Icon, Image } from 'semantic-ui-react';
+
+import { getListings } from '../../redux/actions/listingsActions';
+import ListingCard from '../components/ListingCard';
+
+@connect((store) => {
+	return {
+        listings: store.listingsReducer.listings
+	}
+})
+
+export default class Listings extends React.Component {
+    constructor() {
+        super();
+
+        this.mapListings = this.mapListings.bind(this);
+        this.refreshListings = this.refreshListings.bind(this);
+        this.viewListing = this.viewListing.bind(this);
+    }
+
+    componentWillMount() {
+        this.props.dispatch(getListings());
+    }
+
+    mapListings = (listings) => {
```

Comment:
mapListings = (listings) => listings.map((listing, i) => <ListingCard key = { i } listing = { listing } id = { i } viewListing = { () => this.viewListing(listing) } />)